### PR TITLE
Fix variable reference

### PR DESCRIPTION
**Pull request by kequach**
Variable reference apparently changed from GITHUB_TOKEN to TOKEN. 
image

### CODE REVIEW

**Comment by kequach** [comment]
Hmm, this still doesn't work even with the correct variable reference due to repository policies:
image

I see 3 options here:
1. We fix the pipeline to work with new policy settings
2. We change the policy settings
3. We remove the gh-pages

**Comment by J535D165** [comment]
Thanks for pointing out option 3. The GitHub landing page has improved over the last year and is now easier to read for non-coders. Personally, I'm never using the website.

**Comment by jelletreep** [comment]
Would be fine with option 3.

To fix it, i think you now have to give the token permission to write:
```
permissions:
  contents: write
```
[https://github.com/JamesIves/github-pages-deploy-action](https://github.com/JamesIves/github-pages-deploy-action)

It is also possible to store the website as artifact (instead of using the gh-pages branch):
[https://github.com/actions/deploy-pages](https://github.com/actions/deploy-pages)
In this case you would have to change the repository settings for github pages.

Happy to update the workflow if we don't go for option 3

**Comment by kequach** [comment]
I personally find option 3 sufficient. The readme is totally fine. The main benefit the website had was the additional ToC to navigate, but as Jonathan pointed out, the READMEs on Github improved a lot. You can now navigate in the Readme too: 
image
if you are also fine with that, I'll just remove the website

**Comment by jelletreep** [comment]
Yes, I am totally fine with removing the website!

**Comment by kequach** [comment]
I removed the relevant stuff, should be good to merge

File: .github/workflows/sort_readme.yml
Comment (jelletreep) [inline .github/workflows/sort_readme.yml:1]: Is this workflow meant to sort and update the README.md if it is not properly sorted? If so, then I think we should add a commit action to commit the result

File: .github/workflows/sort_readme.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+          persist-credentials: false
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+      - name: Install Python dependencies
+        run: |
+          pip install -r requirements.txt
+      - name: sort readme # run the run.py to get the latest data
+        run: |
+          python sort.py
```
Comment (jelletreep) [inline .github/workflows/sort_readme.yml:19]:
```suggestion
          python sort.py
          
       - name: Commit files
         run: |
           git config --local user.email "github@uu.nl"
           git config --local user.name "GitHub Utrecht University"
           git add .
           git commit -m "sort README"
        continue-on-error: true
      - name: Push changes
        uses: ad-m/github-push-action@master
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
        continue-on-error: true
          
```

**Review by jelletreep — COMMENTED** [review]
Thanks @kequach for updating the PR!

**Comment by kequach** [comment]
@jelletreep good point, I think the commit/push was implicit with the deployment of the page. Haven't thought about that

**Review by jelletreep — APPROVED** [review]

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks!

**Pull request merged**